### ISSUE TITLE
[feature request] ability to download online subtitles 

### ISSUE THREAD

**Issue by Sensex00** [abdallahmehiz/mpvKt#34]
I want the app to be search and download
Online subtitles from opensubtiles.org like in vlc or mx player

**Comment by virtuacool** [comment]
+1
This feature is so important! I understand English, but it's not my native language. Having subtitles is important for 3/4 of the planet! 👍

**Comment by WybeBosch** [comment]
Iknow that this feature goes a bit outside of staying close to original MPV, but it would be much appriciated!
Right now its the only feature im missing as compared to MX Player

**Comment by Sensex00** [comment]
Yes ability to get subtitle readily from the player itself is a big win also made this similar request here at mpvEx https://github.com/marlboro-advance/mpvEx/issues/180 with recommending wyzie subs api https://docs.wyzie.ru/
